src/services/access.service.ts

```diff
@@ -9,11 +9,22 @@ const AccessService = {
   getAccountsForOrg,
   getDetailedAccountsForOrg,
   getIndividualAccount,
+  isSynapseEmail,
   updateIndividualAccountRoles,
 }
 
-async function getAliasFromSynapseByEmail(synapseEmailAddress: string): Promise<any> {
-  if (synapseEmailAddress.indexOf('@synapse.org') === -1 && synapseEmailAddress.indexOf('@synapse.org') > -1) {
+function isSynapseEmail(emailAddress: string): boolean {
+  return emailAddress.replace(new RegExp('.*@'), '') === 'synapse.org'
```

Comment:
Previous expression always evaluated to false -- update expression to distinguish Synapse vs non-Synapse email addresses

Reply:
Is there a reason you can't use `emailAddress.endsWith('@synapse.org')`? I think a RegEx is overkill here since you're not really doing any validation

Reply:
Good call! I'll update in the next commit